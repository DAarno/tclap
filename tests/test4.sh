#!/bin/sh
# failure
`dirname $0`/simple-test.sh `basename $0 .sh` test1
