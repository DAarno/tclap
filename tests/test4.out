PARSE ERROR:  
             Required argument missing: name

Brief USAGE: 
   ../examples/test1  [-hr] [--version] -n <string>

For complete USAGE and HELP type: 
   ../examples/test1 --help

